src/potFormater.js

```diff
@@ -1,20 +1,55 @@
 /**
+ * Ensure that multi-line strings are properly commented out
+ *
+ * For instance:
+ * This is\nmy multi-line\ncomment
+ *
+ * should be escaped as :
+ * #. This is
+ * #. my multi-line
+ * #. comment
+ *
+ * @param {String} commentPrefix
+ * @param {String} rawComment
+ * @returns {String}
+ *
+ * @author Guillaume Boddaert
+ */
+const potCommentMultiLineWrapper = (commentPrefix, rawComment) => {
+  const comments = rawComment.split('\n');
+  return comments.reduce((a, b) => `${a}${commentPrefix} ${b}\n`, '');
+};
+
+
+/**
  * Formatting POT comments
- * @param {Object[]}
+ * @param {Object[]} messageList
  * @return {String}
  *
  * example: see tests
  *
  * @author Michael Hsu
+ * @author Guillaume Boddaert
  */
 const potCommentsFormater = messageList =>
-  messageList.reduce((acc, { filename, id, description }) =>
-    `${acc}#: ${filename}\n#. [${id}] - ${description}\n`
-  , '');
+  messageList.reduce((acc, { filename, id, description, defaultMessage, mappedBy }) => {
+    let out = acc;
+    out += potCommentMultiLineWrapper('#:', filename);
+    if (description) {
+      out += potCommentMultiLineWrapper('#.', `[${id}] - ${description}`);
+    } else {
+      out += potCommentMultiLineWrapper('#.', `[${id}]`);
+    }
+    if (mappedBy && mappedBy !== 'defaultMessage') {
+      out += potCommentMultiLineWrapper('#.', `defaultMessage is:\n${defaultMessage}`);
```

Comment:
As I mention at the [issue](https://github.com/evenchange4/react-intl-po/issues/57#issuecomment-276321358).
How about just adding `defaultMessage comment` for all case? The `mappedBy` can be removed to simplify the codebase because it is uncontrollable by the consumer of react-intl-po.